docker/dev/run-spec.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/bash
-# use:  docker-compose run --rm app docker/dev/run-spec.sh
-# Run rspec tests in docker environment.
 #
+# Run rspec tests in docker environment:
+#   – Execute like this: `docker-compose run --rm app ./docker/dev/run-rspec.sh`
+#   – Or make an alias `alias dspec='docker-compose run --rm app ./docker/dev/run-rspec.sh'`
+#        then type `dspec` to start Continuous Integration Testing.
+#   – Or run from shell in docker (`docker-compose run --rm bash` … ./docker/dev/run-rpsec.sh`)
 
 #
 # Prepare spec tests
 #
-RAILS_ENV=test bundle exec rake db:test:prepare
+export RTAILS_ENV=test
```

Comment:
Same as above: should this be RAILS_ENV rather than RTAILS_ENV?
I found this same typo in some changes to the rigse repo. I fixed it in a branch for a bug fix, but that branch has not been merged yet. I think this destroys your dev database.